src/System.ComponentModel.Annotations/tests/DisplayFormatAttributeTest.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations.Tests
+{
+    public class DisplayFormatAttributeTest
```

Comment:
Add a separate theory that leaves `NullDisplayText` untouched and confirms `GetNullDisplayText()` returns `null` when `NullDisplayTextResourceType` is either `null` or `typeof(FakeResourceType)`.